dashboard/client/requests_over_time/requests_over_time.js

```diff
@@ -0,0 +1,114 @@
+import d3 from 'd3';
+import nvd3 from 'nvd3';
+import _ from 'lodash';
+
+Template.requestsOverTime.onRendered(function () {
+  // Get reference to template instance
+  const templateInstance = Template.instance();
+
+  templateInstance.elasticsearchData = new ReactiveVar();
+  templateInstance.chartData = new ReactiveVar();
+
+  const queryParams = {
+    size: 0,
+    body: {
+      query: {
+        filtered: {
+          query: {
+            bool: {
+              should: [
+                {
+                  wildcard: {
+                    request_path: {
+                      // Add '*' to partially match the url
+                      value: '/*',
+                    },
+                  },
+                },
+              ],
+            },
+          },
+        },
+      },
+      aggs: {
+        requests_over_time: {
+          date_histogram: {
+            field: 'request_at',
+            interval: 'month',
+            format: 'yyyy-MM-dd'
+          },
+        },
+      },
+    },
+  };
+
+  // Initialize chart
+  const chart = nvd3.models.historicalBarChart();
+
+  // configure chart
+  chart
+    .x(d => d.key)
+    .y(d => d.doc_count)
+    .xScale(d3.time.scale())
+    .margin({left: 100, bottom: 100})
+    .showXAxis(true);
+
+  // configure x-axis settings for chart
+  chart.xAxis
+    .axisLabel('Days')
+    // Format dates in m/d/y format
+    .tickFormat(d => d3.time.format('%x')(new Date(d)));
+
+  // configure y-axis settings for chart
+  chart.yAxis
+    .axisLabel('Requests');
+
+  // Fetch Elasticsearch data reactively
+  templateInstance.autorun(function () {
+    const elasticsearchHost = Template.currentData().elasticsearchHost;
+
+    if (elasticsearchHost) {
+      // Get Elasticsearch data
+      Meteor.call('getElasticsearchData', elasticsearchHost, queryParams, function (error, result) {
+        // Update Elasticsearch data reactive variable with result
+        templateInstance.elasticsearchData.set(result);
+      });
+    }
+  });
+
+  // Parse chart data reactively
+  templateInstance.autorun(function () {
+    const elasticsearchData = templateInstance.elasticsearchData.get();
+
+    if (elasticsearchData) {
+      // Get aggregations from Elasticsearch data
+      const aggregatedData = elasticsearchData.aggregations.requests_over_time.buckets;
+
+      const chartData = [
+        {
+          key: "Requests over time",
+          values: aggregatedData
+        }
+      ];
+
+      // Update chart data reactive variable
+      templateInstance.chartData.set(chartData);
+    }
+  });
+
+  // Render chart reactively
+  templateInstance.autorun(function () {
+    // Get chart data from reactive variable
+    const chartData = templateInstance.chartData.get();
+
+    if (chartData) {
+      // Render the chart with data
+      d3.select('#requests-over-time-chart svg')
+        .datum(chartData)
```

Comment:
Can we work on the chart size in a different PR? I would like to continue working on the wireframes, and we might need to adjust the chart sizes once we have more template structure in place.